api-models/thrift/src/main/thrift/appsRendering.thrift

```diff
@@ -3,6 +3,19 @@ include "content/v1.thrift"
 namespace scala com.gu.mobile.ar.models
 namespace typescript _at_guardian.apps_rendering_api_models
 
+struct Image {
+    1: required string url
+    2: required i32 height
+    3: required i32 width
```

Comment:
What do we use these for in Apps-Rendering?

Reply:
Nothing at the moment, I thought they might be useful to know the aspect ratio but I'm not sure if they are required in the image resizing code?

Reply:
> I thought they might be useful to know the aspect ratio

Sounds sensible 👍